src/client/pages/EmailSent.tsx

```diff
@@ -110,21 +112,22 @@ export const EmailSent = ({
           >
             <EmailInput defaultValue={email} hidden hideLabel />
           </MainForm>
-          <MainBodyText cssOverrides={belowFormMarginTopSpacingStyle}>
-            If you are still having trouble, contact our customer service team
-            at{' '}
-            <ExternalLink
-              cssOverrides={css`
-                font-weight: 700;
-              `}
-              href="mailto:userhelp@theguardian.com"
-            >
-              userhelp@guardian.com
-            </ExternalLink>
-            .
-          </MainBodyText>
         </>
       )}
+      {(showHelp || (email && resendEmailAction)) && (
```

Comment:
Move this into a variable before the `return` statement

```suggestion
const showHelpBox = showHelp || (email && resendEmailAction);
...
      {showHelpBox && (
```